listView.php

```diff
@@ -249,6 +245,18 @@
 $stmt = $db->prepare('SELECT * FROM taskList WHERE boardID = ? ORDER BY `listID` ASC');
             $stmt->execute(array($_GET['board']));       
             $lists = $stmt->fetchAll();
+if ($stmt->rowCount() == 0) {
+  //There is no lists, show the user an "empty board" message
+?>
+<div class="row">
+	<div class="jumbotron">
+		  <h1 class="display-4">Empty Board</h1>
+		  <p class="lead">The board you have selected is empty. Click "Create List" to get started</p>
```

Comment:
Good to see that the instruction was included - makes the application more user-friendly.